conftest.py

```diff
@@ -196,8 +211,7 @@ def run_webworker(self, code):
         return self.run_js(
             """
             let worker = new Worker( '{}' );
```

Comment:
Would we perhaps want to make `selenium_webworker_standalone` start a webworker and then have all `run_webworker` uses run in the same webworker? It's a little surprising to me that each call to `run_webworker` creates it's own `Worker` instance.

Reply:
Yep, I also found the current behavior surprising and +1 to move in that direction.

Though for now, I'm only looking into fixing CI and this refactoring can be done separately. Currently we are always calling `run_webworker` only once per test, so it should be fine.

Reply:
> Any ideas what could lead to, `JavascriptException: _createPyodideModule is not defined`

No clue! Maybe a problem loading `pyodide.asm.js` which is where that is defined...